### ISSUE TITLE
FileList not working (again)

### ISSUE THREAD

**Issue by dragosmiron** [Jackett/Jackett#768]
Hello,
For FileList, I am able to search for torrents, but they don't download

here are the logs
```
Nov 29, 2016 11:14:34 AM	Info	
ApiControllerActionInvoker InvokeActionAsync 
Nov 29, 2016 11:14:34 AM	Info	
ReflectedHttpActionDescriptor ExecuteAsync Action returned 'StatusCode: 404, ReasonPhrase: 'Not Found', Version: 1.1, Content: <null>, Headers:
{
}'
Nov 29, 2016 11:14:34 AM	Error	
Error downloading filelist http://filelist.ro/
Nov 29, 2016 11:14:34 AM	Debug	
UnixLibCurlWebClient:GetBytes Returning OK => 23285 bytes
Nov 29, 2016 11:14:33 AM	Debug	
UnixLibCurlWebClient:GetBytes(Url:http://filelist.ro/)
```
ref https://github.com/Jackett/Jackett/issues/765

**Comment by dragosmiron** [comment]
@vacantrogue i wouldn't have opened this if it was working no?

**Comment by dragosmiron** [comment]
I can confirm I tested from v0.7.653 and is not working. The same on 0.7.335

**Comment by dragosmiron** [comment]
I can make a video if you want

**Comment by dragosmiron** [comment]
I can also provide you with an invite

**Comment by dragosmiron** [comment]
tagging @kaso17

**Comment by dragosmiron** [comment]
If it is fixed, and in master, does it mean we have to wait for a release then ? I can see was merged 1 day ago

**Comment by kaso17** [comment]
Just made a release containing the fix (v0.7.673)
